packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -151,6 +151,12 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	@Input()
 	public data: object[] = [];
 
+	/**
+	 * Data that will be display inside your table.
+	 */
+	@Input()
+	public mode: "default" | "compact" = "compact";
```

Comment:
I totally agree, but could we do it in a separated issue ? 
Because this feature is required for Bessy and it's quite urgent :-(
Or do you think there's a way they can use  the custom pagination on their side without changing anything in Stark ?

Reply:
I'm afraid that this is actually the point. By implementing this change as it is now we are kind of hacking the way the components currently work and we'll be only adding more code that will be dropped afterwards (hopefully... Or it may be kept there for a long time if we've no time for maintain it).

The solution I proposed is not such a big change, in the sense that it is not a breaking change because we'll simply deprecate some options but not remove them. We just need to add some options and add some logic to give them preference over the "deprecated" ones...

@SuperITMan what do you think it should be done?

Reply:
okay, I agree, it's the all component that is messy, because we also have bugs with the pagination when using the filter. It's not the topic but it could be grate to solve it too

Reply:
I think you could add the extra input for now and afterwards actually make a "compact" mode for the table